From 707d69bf1e7a5f2e29309239d599933350cbfd59 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Wed, 4 Jan 2023 08:04:07 +0100
Subject: [PATCH] tests: musl doesn't seem to recognize UTC0 as a timezone
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

UTC works though.

TODO: Figure out if musl /should/ regonize UTC0.
---
 testsuite/date/date-timezone | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/testsuite/date/date-timezone b/testsuite/date/date-timezone
index 8628aa1d7..720ce4f26 100644
--- a/testsuite/date/date-timezone
+++ b/testsuite/date/date-timezone
@@ -1,17 +1,17 @@
 # FEATURE: CONFIG_FEATURE_TIMEZONE
 
 # 'Z' is UTC
-dt=$(TZ=UTC0 busybox date -d '1999-1-2 3:4:5Z')
+dt=$(TZ=UTC busybox date -d '1999-1-2 3:4:5Z')
 dt=$(echo "$dt" | cut -b1-19)
 test x"$dt" = x"Sat Jan  2 03:04:05"
 
 # '+0600' is six hours ahead of UTC
-dt=$(TZ=UTC0 busybox date -d '1999-1-2 3:4:5 +0600')
+dt=$(TZ=UTC busybox date -d '1999-1-2 3:4:5 +0600')
 dt=$(echo "$dt" | cut -b1-19)
 test x"$dt" = x"Fri Jan  1 21:04:05"
 
 # '-0600' is six hours behind UTC
-dt=$(TZ=UTC0 busybox date -d '1999-1-2 3:4:5 -0600')
+dt=$(TZ=UTC busybox date -d '1999-1-2 3:4:5 -0600')
 dt=$(echo "$dt" | cut -b1-19)
 test x"$dt" = x"Sat Jan  2 09:04:05"
 
